### PR TITLE
Always format postal number with space

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

File: src/pages/Chat/steps/AddressInput.tsx [hunk 1]
```diff
@@ -116,19 +118,25 @@ export const AddressInput: React.SFC<AddressInputProps & Focusable> = ({
                     <TranslationsConsumer textKey="CHAT_INPUT_ADDRESS_POSTAL_NUMBER_PLACEHOLDER">
                       {(placeholder) => (
                         <UserTextInput
-                          type="number"
+                          type="text"
```
Comment (hedvigoscar) [inline src/pages/Chat/steps/AddressInput.tsx]: how about we use `type="tel"` here? Then we will get the nice number keyboard despite allowing other characters.

I know it's non-semantic and a bit hacky though 🙁

Comment (palmfjord) [reply src/pages/Chat/steps/AddressInput.tsx]: u sure? i don't think it does :/

Comment (hedvigoscar) [reply src/pages/Chat/steps/AddressInput.tsx]: it's possible that I'm just imagining it 😞

Comment (hedvigoscar) [reply src/pages/Chat/steps/AddressInput.tsx]:
```suggestion
                          type="text"
                          inputMode="numeric"
```

Comment (hedvigoscar) [reply src/pages/Chat/steps/AddressInput.tsx]: We'll save this for later.

**Pull request merged**